test/Deployment.js

```diff
@@ -6,77 +8,77 @@ const config = {
         {
             symbol: 'XXX',
             decimals: 18,
-            supply: '1829101'
+            supply: 1_829_101
         },
         {
             symbol: 'YYY',
             decimals: 18,
-            supply: '3603801'
+            supply: 3_603_801
         },
         {
             symbol: 'BNT',
             decimals: 18,
-            supply: '6914855'
+            supply: 6_914_855
         },
         {
             symbol: 'vBNT',
             decimals: 18,
-            supply: '0'
+            supply: 0
         }
     ],
     converters: [
         {
             type: 3,
             symbol: 'ETHBNT',
             decimals: 18,
-            fee: '0.1%',
+            fee: 10_000,
```

Comment:
makes sense for supply, but why here? why not user readable values?

Reply:
I thought that it's easier to handle than a textual `0.1%`, but I can bring it back. Perhaps `0.1` (instead of `0.1%`)?

Reply:
sure, 0.1 is fine

Reply:
I've restored "%" as well so that it'd clearer that it's 0.1% and not 0.1 (10%).

Reply:
What's wrong with the way that I've handled `%` in the original version of the deployment script?
```
const percentageToPPM = (value) => {
    return decimalToInteger(value.replace('%', ''), 4);
};
```

Reply:
Parsing numbers as strings as a bit buggy. For example, `percentageToPPM("0.123456%")` would have returned `123,456`